root/components/Table.js

```diff
@@ -12,7 +12,7 @@ import {useTable} from 'react-table';
 
 import loopParity from '../utility/loopParity';
 
-const Table = (({columns, data}) => {
+const Table = (({columns, data}: any) => {
```

Comment:
I'd really really like these not to be any, if possible.

Reply:
See https://github.com/metabrainz/musicbrainz-server/pull/1246#issuecomment-551162606